index.html

```diff
@@ -28,13 +28,20 @@
   <header class="header">
     <div class="toolbar logo">Ghilain Ishimwe</div>
     <nav class="toolbar imagcss">
-      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" class="logo-text"
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" id="pop" height="24" class="logo-text"
                         viewBox="0 0 24 24">
                         <path d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z" />
                     </svg>
-      <a href="#contact" class="contactlink">Contact</a> 
-      <a href="#about" class="aboutlink">About</a>
-      <a href="#portfolio" class="portfolink">Portfolio</a>
+                    <div class="nav-list">
+                      <span class="close-x">&times;</span>
+                      <a href="#contact" class="contactlink">Contact</a> 
+                      <div class="borderlink borderlink1"></div>
+                      <a href="#about" class="aboutlink">About</a>
+                      <div class="borderlink borderlink2"></div>
+                      <a href="#portfolio" class="portfolink">Portfolio</a>
```

Comment:
- [ ] Your "Portfolio" link does not link to the correct "recent-work" section.  It links to your headline.  Kindly correct this link so we can move closer to approval 💪